app/src/App.js

```diff
@@ -1,25 +1,23 @@
-import styled from "styled-components";
-import { Header } from "./components/Header";
-import { Content } from "./components/Content";
-import { InputElement } from "./components/InputElement";
-import { GlobalStyles } from "./utils/GlobalStyles";
+import { StyledWrapper } from "./App.styles";
+import { Header } from "./components/Header/Header";
+import { Content } from "./components/Content/Content";
+import { InputElement } from "./components/InputElement/InputElement";
+import { GlobalStyles } from "./GlobalStyles";
+import { ThemeProvider } from "styled-components";
+import { theme } from "./Theme";
+import Blank from "./";
 
 export const App = () => {
   return (
     <>
-      <GlobalStyles />
-      <Wrapper>
-        <Header />
-        <Content/>
-        <InputElement/>
-      </Wrapper>
+      <ThemeProvider theme={theme}>
```

Comment:
This is exactly how to do this. Good job!

Reply:
thanks :)